libs/agno/agno/agent/agent.py

```diff
@@ -1551,6 +1551,8 @@ def run(
         videos: Optional[Sequence[Video]] = None,
         files: Optional[Sequence[File]] = None,
         retries: Optional[int] = None,
+        delay_between_retries: Optional[int] = None,
+        exponential_backoff: Optional[bool] = None,
```

Comment:
I actually think this makes the `run` too bloated. I don't see why you'd want to set it on run to start with. I think we should deprecate `retries` on run.